corehq/apps/style/static/style/lib/bootstrap/less/accordion.less

```diff
@@ -32,3 +32,8 @@
   padding: 9px 15px;
   border-top: 1px solid #e5e5e5;
 }
+
+//When there are help popovers within small accordions, the popovers are truncated
+.accordion-body.in {
+  overflow:visible;
+}
```

Comment:
I don't think we're supposed to modify this file. I believe this change should go in https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/static/style/less/components/accordion.less cc @biyeun to confirm